src/font/face.zig

```diff
@@ -100,6 +100,15 @@ pub const RenderOptions = struct {
     ///
     /// This only works with CoreText currently.
     thicken: bool = false,
+
+    /// "Strength" of the thickening, between `0` and `255`.
+    /// Only has an effect when `thicken` is enabled.
+    ///
+    /// `0` does not correspond to *no* thickening,
+    /// just the *lightest* thickening available.
+    ///
+    /// CoreText only.
+    thicken_strength: u8 = 255,
```

Comment:
My only thought here is should we use a f32 here instead of a u8. It feels like this would be more user-friendly to configure for a range of "min" to "max".

Reply:
I considered it but it felt like false precision to me-- I'm not particularly bothered about it either way though so I'm fine with changing it to a float if you want.

Reply:
I can see the argument, but the argument types match and the difference between subtle changes should not be noticeable enough.

Reply:
I take it back, this causes hashing issues and I don't want to deal with it lol.